lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -96,27 +99,27 @@ public function get_font_collection( $request ) {
 	 *
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
-	public function get_font_collections() {
+	public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Minor nitpick: `get_items` is usually above `get_item` in the file

Reply:
moved